### ISSUE TITLE
Local audio playback issue about every 10s over BT

### ISSUE THREAD

**Issue by PJansky** [GrapheneOS/os-issue-tracker#3169]
I'm having some minor audio glitches around every 10s when playing local audio files. A am able to reproduce the issue with multiple audio players (Metro, Oto and Nyx) and multiple Bluetooth devices (Sony WF-1000XM4 and Samsung Galaxy Buds+) and therefore believe it to be an underlying issue rather than a player issue. It could still be a BT issue but I haven't tested that yet. It sounds like some buffer underrun/overrun after which playback normally continues. I also suspected some battery optimization could cause a miss of some deadline, but I've already tried fiddling around with battery optimization settings to no avail. I will test it further after my exams. I don't now if this is a new regression or not as I've only started using Graphene 2 weeks ago and it has been present since then. It has otherwise been rock solid, thanks for your amazing work :)

Device: Pixel 8 Pro
Build: UQ1A.240105.004.2024102600
Installation Method:  Webinstaller after using stock OS

**Comment by LunchBoxC** [comment]
I opened a Bluetooth issue on here regarding my Bluetooth headset where it would start distorting the audio after a period of time. I messed around with the developer options and tried all sorts of stuff including a reset of the phone and reinstalling stock Google OS and also Graphene again and still the issue persists.  My issue on here was closed as "Upstream" bug meaning an issue with the AOSP.  There are numerous issues around Bluetooth and scores of people complaining to Google about my issue and your type of issue as well.  I just wanted to let you know I think it's a big problem at Google affecting Google Pixel 6-8's for some reason.  Google keeps locking the threads of people talking about it so they are either covering up something and denying the problem or have a fix in the works.  Just thought I'd let you know.  Mine has become so bad I've had to start using a USB C to wired Headset so I can at least use my phone in my wheelchair because I can't hold the phone while wheeling around.  It has me questioning whether or not I should give up and go to something else because I'm so fed up with all the issues.  (Not Graphene's fault but rather google and their pushing out of Android 14 before it was fully ready).  I had 0 issues on Android 13 and I have 0 issues with my old Pixel 5.  Just for what it's worth.  Good luck!
